src/modules/displayList.js

```diff
@@ -0,0 +1,63 @@
+import { taskarr } from './addTask.js';
+import '../assets/trash-can.png';
+import { saveData } from './userInput.js';
+
+export const tasksList = document.querySelector('#tasksList');
+
+export default (task) => {
+  for (let i = 0; i < taskarr.length; i += 1) {
+    task = document.createElement('li');
+    task.classList.add('newTask');
+
+    if (taskarr[i].description !== '' && taskarr[i].completed === true) {
+      task.innerHTML = `
+      <input type="checkbox" checked class="checkB" ${taskarr[i].completed} autocomplete="false"  />
+      <input class="newTasks" type="text" id="addItem" value="${taskarr[i].description}" />
+      <img class="trash" id="trash"  src="./assets/trash-can.png" alt="" />
+   
+      `;
+    }
+    if (taskarr[i].description !== '' && taskarr[i].completed === false) {
+      task.innerHTML = `
+      <input type="checkbox" class="checkB" ${taskarr[i].completed}  />
+      <input class="newTasks" type="text" id="addItem" value="${taskarr[i].description}" />
+      <img class="trash" id="trash"  src="./assets/three-dots.png" alt="" />
+      `;
+    }
+
+    if (taskarr[i].description === '') {
+      tasksList.innerHTML = '';
+    }
+
+    tasksList.appendChild(task);
+  }
+
+  const taskDescription = document.querySelectorAll('#addItem');
+
+  taskDescription.forEach((task, index) => {
+    task.addEventListener('click', (event) => {
+      task.classList.add('edit');
+      // if (event === false) {
+      //   taskDescription.parentElement.addEventListener('click', () => task.classList.remove('edit'));
+      // }
```

Comment:
Kindly removing commented lines of code in your project. It makes your code messy and difficult to read